app/assets/stylesheets/components/_theme.css.scss

```diff
@@ -139,6 +139,10 @@ md-input-group {
   }
 }
 
+.md-padding {
+  padding: 16px;
```

Comment:
I'm using it [here](https://github.com/groupbuddies/council/blob/63e3379d7d6db6ceb29e4b72f2d26f66a4173002/app/assets/javascripts/app/discussions/templates/discussion.html.slim#L3)